### PR TITLE
Update BMO default branch in build_ipa

### PR DESCRIPTION
**Pull request by mboukhalfa**
Default branch was changed from master to main in BMO, this PR updates the script build_ipa to consider the change

### CODE REVIEW

File: ci/scripts/image_scripts/build_ipa.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -eu
 
 # Repository configuration options
 METAL3_DEV_ENV_REPO="https://github.com/metal3-io/metal3-dev-env"
-METAL3_DEV_ENV_BRANCH="${METAL3_DEV_ENV_BRANCH:-master}"
+METAL3_DEV_ENV_BRANCH="${METAL3_DEV_ENV_BRANCH:-main}"
```
Comment (kashifest) [inline ci/scripts/image_scripts/build_ipa.sh:8]: this is not main yet

**Review by kashifest — DISMISSED** [review]

**Review by fmuyassarov — DISMISSED** [review]

**Comment by mboukhalfa** [comment]
@kashifest @fmuyassarov Is it ok to keep the branch name  `Update-dev-env-and-bmo-default-branch/mohammed` or I need to close and create a new branch with proper name since we are not changing dev-env default branch in build_ipa script

**Comment by kashifest** [comment]
> @kashifest @fmuyassarov Is it ok to keep the branch name `Update-dev-env-and-bmo-default-branch/mohammed` or I need to close and create a new branch with proper name since we are not changing dev-env default branch in build_ipa script

Branch name shouldn't matter much as long as commit message is correct

**Review by fmuyassarov — APPROVED** [review]
Thanks @mboukhalfa 
@kashifest we will need similar changes for other repos too once switched to main. Just to remember.

**Pull request merged**